crest.md

```diff
@@ -126,7 +126,7 @@ The loader exposes its current configuration in the local window, by setting a *
 
 - **config**: Contains configurations for Crest's operation.
 
-  - **crestConfig**: Specific configurations for Crest's behavior.
+  - **crestConfig**: Specific configurations for Crest's behaviour.
```

Comment:
Wasn't the agreement to write in US English?  
Not saying to go through the entire repo, but keep it in mind going forward.

Reply:
I would like at least the documentation to be consistent with the Dashboard - if one is US English, the other should be too.

The release notes I'm fine with being in UK English.

Reply:
I also changed the other instance in this file that has been British for forever. There were no further behavio**u**rs in this repo that I could find.